.github/workflows/build.yml

```diff
@@ -68,7 +69,8 @@ jobs:
             ./gradlew build
             ${{ matrix.coverage && 'jacocoTestReport' || '' }}
             -PtestJavaVersion=${{ matrix.test-java-version }}
-            -Porg.gradle.java.installations.paths=${{ steps.setup-java-test.outputs.path }},${{ steps.setup-java.outputs.path }}
+            "-Porg.gradle.java.installations.paths=${{ steps.setup-java-test.outputs.path }}"
+            "-Porg.gradle.java.installations.auto-download=false"
```

Comment:
I think the double quotes ended up being the only thing that was needed here, but the other changes seem good and align with the opentelemetry-java-instrumentation project (where, btw, we only run smoke tests against windows but not regular integration or unit tests, so haven't run into this double quote issue)